cli/tools/lint/linter.rs

```diff
@@ -97,6 +99,15 @@ impl CliLinter {
     ext: Option<&str>,
   ) -> Result<(ParsedSource, Vec<LintDiagnostic>), AnyError> {
     let specifier = specifier_from_file_path(file_path)?;
+
+    let metrics = minified_file::analyze_content(&source_code);
+    if metrics.is_likely_minified() {
+      bail!(
+        "{} appears to be a minified file, skipping linting",
+        specifier.as_str()
+      );
+    }
```

Comment:
This should be a "warning" lint diagnostic. We should probably have a way to opt-in into linting files if they are specified as `deno lint foo.min.js`, but that's gonna be hard since there's no discrimination between auto-discovered files and ones passed explicitly on the CLI